recipe/meta.yaml

```
@@ -19,13 +20,13 @@ requirements:
   host:
     - python
     - pip
-    - numpy {{ numpy }}
+    - numpy >=2.0.0
     - setuptools
     - wheel
     - versioneer
   run:
     - python
-    - {{ pin_compatible('numpy') }}
+    - numpy >=2.0.0
```

```suggestion
    - numpy
```
On top of Serhii's recommendation, building with numpy 2 brings compatiblity with odler numpy. There is a run_exports on numpy. too.